base/nullable.jl

```diff
@@ -63,16 +63,79 @@ get(x::Nullable) = x.isnull ? throw(NullException()) : x.value
 
 isnull(x::Nullable) = x.isnull
 
-function isequal(x::Nullable, y::Nullable)
-    if x.isnull && y.isnull
-        return true
-    elseif x.isnull || y.isnull
-        return false
+
+## Operators
+
+"""
+    null_safe_op(f::Any, ::Type)::Bool
```

Comment:
`null_safe_op` is fine by me as well. In math terms, this is basically an assertion that an operation is uniformly safe over the space of all bit patterns, but I think that term is too mathematic to make sense in our codebase.